src/Components/Auth/Auth.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useState } from "react";
+import TextField from "./TextField";
+import "./auth.css";
+import { NavLink } from "react-router-dom";
+
+const Auth = (props) => {
+  const [auth, setAuth] = useState({});
+  const type = props.type;
+  const onChange = (e) => {
+    setAuth((prev) => {
+      [e.target.name] = e.target.value;
+    });
+  };
+  return (
+    <section id="auth">
+      <div className="container">
+        <div className="auth-form-title">
+          <h1>{props.type}</h1>
+        </div>
+        <form id="auth-form">
+          <TextField
+            type="email"
+            placeholder="Email"
+            onChange={onChange}
+            name="email"
+          />
+          <TextField
+            type="password"
+            placeholder="Password"
+            onChange={onChange}
+            name="password"
+          />
+          {type === "Sign Up" && (
+            <TextField
+              type="password"
+              placeholder="Repeat Password"
+              onChange={onChange}
+              name="RepeatPassword"
+            />
+          )}
+
+          <div className="btn">{type}</div>
```

Comment:
This button wraps the "Sign Up" text for a narrow window such as mobile

Reply:
It's not responsive it. Will work on it.

Reply:
Also, the different button icons for "Sign-up" and "login" are they a design choice? 
At the moment the "sign-up" button doesn't have a white background while the "Login" button does